src/component/AddItem.js

```diff
@@ -46,6 +52,46 @@ const AddItem = ({ firestore }) => {
           className="inputField"
         />
       </label>
+
+      <div className="frequencyButtons">
+        <p>How soon are you likely to buy it again?</p>
+        <input
+          type="radio"
```

Comment:
the buttons on my end are not appearing as radio buttons on my end, is there a specific reason you chose radio? totally works fine just curious

Reply:
So they're radio buttons in that they behave like a radio button, but with the CSS styling, they look like normal buttons! Radio buttons are nice because you limit the user to one choice and you can style it to show which choice they made.

Reply:
Yes, radio buttons allow user to only select one option and they hold information rather than triggering an action. I had originally coded this out with regular buttons, but the radio buttons make the code much simpler!